pass-indexer-core/src/main/resources/esindex.json

```diff
@@ -102,8 +109,10 @@
         "source": {"type": "keyword"},        
         "startDate": {"type": "date"},
         "submission": {"type": "keyword", "normalizer": "fedora_uri"},
+        "submissionStatus": {"type": "keyword"},
         "submitted": {"type": "boolean"},
         "submittedDate": {"type": "date"},
+        "submitter": {"type": "keyword", "normalizer": "fedora_uri"},
```

Comment:
This can be a mailto: URI, which is technically way different than a Fedora URI.  based on looking at the indexer normalizer, I _think_ they'll work fine.  It might be wise to add a test to make sure a record containing a `mailto:` placeholder does not blow up, or test for that in the `java-fedora-client` code.   So it's your call whether to test for that here, or there